docs/types.rst

```diff
@@ -4,74 +4,75 @@ PyOTA Types
+    trytes_2 = TryteString('LH9GYEMHCF9GWHZFEELHVFOEOHNEEEWHZFUD')
+
+    # Comparison and concatenation:
     if trytes_1 != trytes_2:
       trytes_combined = trytes_1 + trytes_2
 
+    # As dictionary keys:
     index = {
       trytes_1: 42,
       trytes_2: 86,
     }
 
-A ``TryteString`` is an ASCII representation of a sequence of trytes. In
-many respects, it is similar to a Python ``bytes`` object (which is an
-ASCII representation of a sequence of bytes).
-
-In fact, the two objects behave very similarly; they support
-concatenation, comparison, can be used as dict keys, etc.
-
-However, unlike ``bytes``, a ``TryteString`` can only contain uppercase
-letters and the number 9 (as a regular expression: ``^[A-Z9]*$``).
-
 As you go through the API documentation, you will see many references to
-``TryteString`` and its subclasses:
+:py:class:`TryteString` and its subclasses:
 
--  ``Fragment``: A signature or message fragment inside a transaction.
+-  :py:class:`Fragment`: A signature or message fragment inside a transaction.
    Fragments are always 2187 trytes long.
--  ``Hash``: An object identifier. Hashes are always 81 trytes long.
+-  :py:class:`Hash`: An object identifier. Hashes are always 81 trytes long.
    There are many different types of hashes:
--  ``Address``: Identifies an address on the Tangle.
--  ``BundleHash``: Identifies a bundle on the Tangle.
--  ``TransactionHash``: Identifies a transaction on the Tangle.
--  ``Seed``: A TryteString that is used for crypto functions such as
+-  :py:class:`Address`: Identifies an address on the Tangle.
+-  :py:class:`BundleHash`: Identifies a bundle on the Tangle.
+-  :py:class:`TransactionHash`: Identifies a transaction on the Tangle.
+-  :py:class:`Seed`: A TryteString that is used for crypto functions such as
    generating addresses, signing inputs, etc. Seeds can be any length,
    but 81 trytes offers the best security.
--  ``Tag``: A tag used to classify a transaction. Tags are always 27
+-  :py:class:`Tag`: A tag used to classify a transaction. Tags are always 27
    trytes long.
--  ``TransactionTrytes``: A TryteString representation of a transaction
+-  :py:class:`TransactionTrytes`: A TryteString representation of a transaction
    on the Tangle. ``TransactionTrytes`` are always 2673 trytes long.
 
+Let's explore the capabilities of the :py:class:`TryteString` base class.
+
 Encoding
 ~~~~~~~~
 
-.. code:: python
-
-    from iota import TryteString
-
-    message_trytes = TryteString.from_unicode('Hello, IOTA!')
-
-To encode character data into trytes, use the
-``TryteString.from_unicode`` method.
+You may use classmethods to create a :py:class:`TryteString` from ``bytes``,
+``unicode string`` or from a list of ``trits``.
 
-You can also convert a tryte sequence into characters using
-``TryteString.decode``. Note that not every tryte sequence can be
-converted; garbage in, garbage out!
+**from_bytes**
```

Comment:
the only thing it will change is that it will appear bold in the index bar on the left side. I found it nicer to stress out function names there when one just would like to browse them there.